include/geode/model/helpers/aabb_model_helpers.h

```diff
@@ -52,4 +54,36 @@ namespace geode
 
     std::tuple< AABBTree2D, absl::FixedArray< uuid > >
         opengeode_model_api create_surfaces_aabb_tree( const Section& model );
+
+    template < index_t dimension >
+    struct ModelMeshesAABBTree
+    {
+        ModelMeshesAABBTree( index_t nb_components )
+            : meshes_trees_( nb_components ), uuids_( nb_components )
+        {
+            tree_ids_.reserve( nb_components );
+        }
+
+        AABBTree< dimension > model_tree_;
+        absl::FixedArray< AABBTree< dimension > > meshes_trees_;
+        absl::FixedArray< uuid > uuids_;
+        absl::flat_hash_map< uuid, index_t > tree_ids_;
+    };
+    ALIAS_2D_AND_3D( ModelMeshesAABBTree );
+
+    ModelMeshesAABBTree3D opengeode_model_api
+        create_lines_and_meshes_aabb_trees( const BRep& model );
```

Comment:
This is more create_lines_aabb_trees, since you don't create aabb trees for other meshes (idem for the others)